TTK_Login.java

```diff
@@ -1,11 +1,25 @@
-import java.util.Scanner;
-
 public class TTK_Login {
-  TTK_User user = new TTK_User();
+
+  private TTK_User user = new TTK_User();  //anropar TTK_User
   user.getUsername();
   user.getPassword();
 
-//  TTK_Main user = new TTK_User();
-//  TTK_Create user = new TTK_User();
-if (username = username)
+  private boolean verifyUser(String username, String password){
+
+  String Username;
+  String Password;
+
+  Password = "Ramisa";
```

Comment:
Lösenord och anvädarnamn är hårdkodat här. Det ska man skriva in i programmet när det körs. Så det behövs inte här.